GiniVision/Classes/Core/GiniVision.swift

```diff
@@ -20,15 +20,15 @@ import UIKit
      Called when the user has taken a picture or imported a file (image or PDF) from camera roll or document explorer
      */
     
-    @objc optional func didCapture(document: GiniVisionDocument)
+    func didCapture(document: GiniVisionDocument)
     
     /**
      Called when the user has taken an image.
      
      - parameter fileData: JPEG image data including meta information or PDF data
      */
-    @available(*, deprecated)
-    @objc optional func didCapture(_ imageData: Data)
+    @available(*, unavailable, renamed: "didCapture(document:)")
```

Comment:
Breaking change?

Reply:
It is a breaking change, but given the feedback I got regarding this delegate, the best thing is to remove the optional methods when it's possible and make the old ones `unavailable`.

If you deprecate a protocol method, Xcode doesn't complain about it. And it could take more time to realize that the method is missing than just using the Xcode assistant to fix it.